### PR TITLE
update reservoir query on flood risk summary page

### PR DESCRIPTION
**Pull request by Matthew-Collins-Defra**

### CODE REVIEW

File: server/services/riskQuery.js [hunk 1]
```diff
@@ -74,28 +103,62 @@ const runQueries = async (x, y, queries) => {
...
+      return esriRequest(requestOptions)
+        .then((response) => { return processEsriHeaders(response, esriRequestUnits) })
+        .then((result) => { return checkResult(result, query) })
     } else {
-      return riskData(query.url).then((result) => { return logPerformance(result, query) })
+      return riskData(query.url).then((result) => { return checkResult(result, query) })
     }
   }))
+  esriRequestUnits.difference = esriRequestUnits.highest - esriRequestUnits.lowest
+  console.log(esriRequestUnits)
```
Comment (MickStein) [inline server/services/riskQuery.js]: Just checking if these logs are meant to be here.

File: server/services/riskQuery.js [hunk 2]
```diff
@@ -122,7 +185,19 @@ async function externalQueries (x, y, queries) {
     }
     results.forEach((result, index) => {
       if (queries[index].esriCall) {
-        featureLayers[queries[index].key] = result.features
+        if (!((result.features) || (result.layers))) {
+          console.log('Error: Invalid response for %s', queries[index].key)
+          console.log(result)
```
Comment (MickStein) [inline server/services/riskQuery.js:191]: Just checking if these logs are meant to be here.

File: server/services/processReservoirRisk.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+function getReservoirWetRisk (riskQueryResult) {
```
Comment (MickStein) [inline server/services/processReservoirRisk.js:1]: Could these be condensed into 1 function potentially? Not sure if that would make things more complex to understand though so happy to forgo.

**Review by MickStein — APPROVED** [review]
LGTM, just a couple of comments but just for checking not preventing merge.

File: server/services/riskQuery.js (hunk 2 above)
Comment (Matthew-Collins-Defra) [reply server/services/riskQuery.js:191]: Yes, this one is to log the data that caused the issue in the error logs, as an error is thrown because of this.

**Pull request merged**